modules/howtos/examples/transactions_example.py

```diff
@@ -105,12 +108,16 @@ def txn_example(ctx):
         doc_c = ctx.get(collection, 'doc-c')
         ctx.remove(doc_c)
 
+        # tag::scope-example[]
+        # Added the above tag (scope-example) to ignore this section in the docs for now.
+        # Once the below TODO is addressed we can remove the tag completely.
         # N1QL query
         # @TODO:  clean up txns query options, scope, pos args and named args won't work
```

Comment:
I had issues with the `scope` option in `TransactionQueryOptions`, I got this error:
```
Error: TransactionFailed{<message=AttributeError("'Scope' object has no attribute 'bucket'")>}
```

@thejcfactor and @charlie-hayes are we expecting this not to work then?

Reply:
Jared did mention to me that some of the options in `TransactionsQueryOptions` don't propagate through, best wait for his review.

Reply:
There is a bug there - IIRC the Scope interface changed and I didn't catch the change in the txn code.

Reply:
Got it, okay thanks for letting me know @davidkelly 👍 .

Reply:
@maria-robobug this is the bug: https://issues.couchbase.com/browse/PYCBC-1322  The real fix is spending a bit of time to completely test the options/config stuff.   It takes the python values and shoves it into a c++ object - so testing that (rather than making a bunch of queries to attempt to see if the options work) should find these sort of errors.  I almost just did that initially, but was feeling a bit rushed....  Like usual, you pay for that stuff later :)